config.rb

```diff
@@ -1,6 +1,11 @@
 $LOAD_PATH << File.expand_path('../lib', __FILE__)
 require 'patches/uri'
 
+# Assets
+# Framework requires set themselves up
+require 'bourbon'
+require 'neat'
```

Comment:
`Bundler.require` maybe?

Reply:
I'm only replacing what is missing from the asset pipeline, I'm not sure requiring everything is a good idea.